DigitisingSC/DHSC-public-website
# Feature/dhsc 157  digital skills db

# Features

- Fixing search config split
- Skills CT and skills search index

# Test

- deploy locally
- create dummy skills locally 
- re-index the skills search index
- navigate to `/skills` 
- check the skills search 

Any questions, let me know.
